README.md

```diff
@@ -226,6 +226,23 @@ kubectl rollout restart deployment/che-operator -n $CHE_NAMESPACE
 
 **Note**: This way to configure dashboard is experimental and may be changed.
 
+## che-server API usage
+
+Currently, Dashboard uses the following che-server API:
+
+| Method | Path                            |
+|--------|---------------------------------|
+| GET    | /kubernetes/namespace/provision |
+| POST   | /kubernetes/namespace           |
+| POST   | /factory/resolver/              |
+| POST   | /factory/token/refresh          |
+| GET    | /workspace/settings             |
+| GET    | /oauth                          |
+| GET    | /oauth/authenticate             |
+| GET    | /oauth/callback                 |
```

Comment:
BTW: [che-workspace-client](https://github.com/eclipse-che/che-workspace-client) doesn't support  bouth ```/oauth/callback``` and ```/oauth/authenticate```.
I think we should remove them.
@akurinnoy WDYT?

Reply:
I saw both `/oauth/callback` and `/oauth/authenticate` in network tab. Where do they come from?

Reply:
@tolusha Oh, I forgot about it. We need them for the authorization flow. We need them.

Reply:
@olexii4 those requests are not initiated by the dashboard

Reply:
@olexii4 @akurinnoy folks, but who is doing `/oauth/callback` && `/oauth/authenticate` ?